### ISSUE TITLE
What is the descriptive context response with Solution Values checked, and Solution Volume changes while the Solute Amount is zero?

### ISSUE THREAD

**Issue by terracoda** [phetsims/molarity#134]
The concentration is not changing, so this is not the correct response.
<img width="715" alt="Screen Shot 2019-07-29 at 1 27 46 PM" src="https://user-images.githubusercontent.com/1902116/62045789-91f7e100-b206-11e9-8dc6-2255cd17c878.png">

@Matthew-phet, could you verify the correct response for this scenario when Solution Values is checked, the Solute Amount is zero and the user changes the Solution Volume? 

I think we discussed this at our last meeting, but I am not sure if I made an issue. In addition, I think we discussed providing a quantitative value for Solute Amount in moles, but maybe an identical response works fine. What do you think? Here are 3 options:

After the quantitative volume value (i.e., object response) the context response (i.e., the alert) needs to describe lack of solute and that the solution is water. For example:

**Option 1**
- object response: "0.750 liters"
- context response: "0.000 moles solute. Solution is pure water."

**Option 2**
- object response: "0.750 liters"
- "Zero solute. Solution is pure water."

**Option 3 - same context response as when Solution Values not checked**
- object response: "0.750 liters"
- "No solute. Solution is pure water."

 I personally, prefer Option 2 and Option 3 over Option 1. Option 2 is more "quantitative" than Option 3, and both are nice and simple.

**Comment by Matthew-phet** [comment]
@twant can you add option 2 for this scenario

**Comment by twant** [comment]
@Matthew-phet absolutely. Currently for qualitative descriptions, the alert text is "No solute. Solution is pure water." Would you like me to change this one to "Zero solute. Solution is pure water" to match the quantitative alert text, or keep them different?

**Comment by terracoda** [comment]
@twant, we would like the qualitative alert (context response) to stay the same at: 
- "No solute. Solution is pure water."
And the quantitative alert for the same situation (i.e., going to zero moles of solute) to be: 
- "Zero solute. Solution is pure water."

**Comment by twant** [comment]
Added these in! Assigning back over for review

**Comment by terracoda** [comment]
Awesome @twant. Looks good in both cases Solution Values Checked and Solution Values not checked. Closing.
